tracer/build/_build/Build.VariableGenerations.cs

```diff
@@ -0,0 +1,321 @@
+                Logger.Info(JsonConvert.SerializeObject(matrix, Formatting.Indented));
+                AzurePipelines.Instance.SetVariable("exploration_tests_linux_matrix", JsonConvert.SerializeObject(matrix, Formatting.None));
+            }
+
+            void GenerateSmokeTestsMatrices()
+            {
+                GenerateSmokeTestsMatrix();
+                GenerateSmokeTestsArm64Matrix();
+            }
+
+            static void GenerateSmokeTestsMatrix()
+            {
+                var matrix = new Dictionary<string, object>();
+
+                AddToMatrix(
+                    new (TargetFramework publishFramework, string dockerTag)[]
+                    {
+                (publishFramework: TargetFramework.NET6_0, "6.0-bullseye-slim"),
+                (publishFramework: TargetFramework.NET5_0, "5.0-bullseye-slim"),
+                (publishFramework: TargetFramework.NET5_0, "5.0-buster-slim"),
+                (publishFramework: TargetFramework.NET5_0, "5.0-focal"),
+                (publishFramework: TargetFramework.NETCOREAPP3_1, "3.1-bullseye-slim"),
+                (publishFramework: TargetFramework.NETCOREAPP3_1, "3.1-buster-slim"),
+                (publishFramework: TargetFramework.NETCOREAPP3_1, "3.1-bionic"),
+                (publishFramework: TargetFramework.NETCOREAPP2_1, "2.1-bionic"),
+                (publishFramework: TargetFramework.NETCOREAPP2_1, "2.1-stretch-slim"),
+                    },
+                    installCmd: "dpkg -i ./datadog-dotnet-apm*_amd64.deb",
+                    linuxArtifacts: "linux-packages-debian",
+                    dockerName: "mcr.microsoft.com/dotnet/aspnet"
+                );
+
+                AddToMatrix(
+                    new (TargetFramework publishFramework, string dockerTag)[]
+                    {
+                (publishFramework: TargetFramework.NET6_0, "34-6.0"),
+                (publishFramework: TargetFramework.NET5_0, "35-5.0"),
+                (publishFramework: TargetFramework.NET5_0, "34-5.0"),
+                (publishFramework: TargetFramework.NET5_0, "33-5.0"),
+                (publishFramework: TargetFramework.NETCOREAPP3_1, "35-3.1"),
+                (publishFramework: TargetFramework.NETCOREAPP3_1, "34-3.1"),
+                (publishFramework: TargetFramework.NETCOREAPP3_1, "33-3.1"),
+                (publishFramework: TargetFramework.NETCOREAPP3_1, "29-3.1"),
+                (publishFramework: TargetFramework.NETCOREAPP2_1, "29-2.1"),
+                    },
+                    installCmd: "rpm -Uvh ./datadog-dotnet-apm*-1.x86_64.rpm",
+                    linuxArtifacts: "linux-packages-debian",
+                    dockerName: "andrewlock/dotnet-fedora"
+                );
+
+                void AddToMatrix(
+                    (TargetFramework publishFramework, string dockerTag)[] images
+                  , string installCmd
+                  , string linuxArtifacts
+                  , string dockerName
+                )
```

Comment:
Understand the sentiment, but prob best to follow existing conventions:
```suggestion
                void AddToMatrix(
                    (TargetFramework publishFramework, string dockerTag)[] images,
                    string installCmd,
                    string linuxArtifacts,
                    string dockerName)
```

Reply:
Fixed in 60573879f2107ca382f238e12f836e05b35338be